SinaiStCatherine/SinaiEt006.xml

```diff
@@ -146,7 +146,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                       <title type="complete" ref="LIT2509Weddas"/>
                       <textLang mainLang="gez"/>
                       <note>The version with the part for Sunday
-                        (sanbata krestiyan) placed the first.</note>
+                        (sanbata krǝstiyān) placed first.</note>
 
```

Comment:
instead of the note one could also list 
LIT2509Weddas#Sunday
LIT2509Weddas#Monday
LIT2509Weddas#Tuesday
etc

Reply:
Frankly speaking, I do not know if the parts are identical. We will  have  a closer look it it later.